pkg/rslog/README.md

```diff
@@ -2,7 +2,8 @@
 
 ## Description
 
-A logging library that meets the requirements of the RStudio logging standards.
+A logging library that meets the requirements of the [Posit logging standards](
+https://rstudiopbc.atlassian.net/wiki/spaces/ENG/pages/36048281/RStudio+Logging+Standard).
```

Comment:
This is a public repo. Should we reference an internal Confluence page? It may be useful for internal users, but it's also unfortunate for those with no access. Maybe we should consider adding another README here that summarizes the standards.